src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/EnvelopeResponseMapperTest.java

```diff
@@ -15,18 +16,21 @@
     private Envelope envelope;
 
     @Before
-    public void setUp() throws Exception {
+    public void setUp() {
         envelope = EnvelopeCreator.envelope();
     }
 
     @Test
-    public void should_map_envelope_to_envelope_response() throws Exception {
+    public void should_map_envelope_to_envelope_response() {
         EnvelopeResponse response = EnvelopeResponseMapper.toEnvelopeResponse(envelope);
 
         assertThat(response)
             .usingComparatorForFields(
                 new ToStringComparator<Classification>(), new String[]{"classification"}
             )
+            .usingComparatorForFields(
```

Comment:
Is this needed?

Reply:
yes. response is mapped to string value whereas assertion is done against domain model which has enum there and assertion does not know how to convert to string automatically. same treatment as `classification` here